distribution/src/assembly/assembly-descriptor-slim.xml

```diff
@@ -128,6 +128,13 @@
             <destName>hazelcast-security-hardened.xml</destName>
             <outputDirectory>config/examples</outputDirectory>
         </file>
+
+        <!-- NOTICE file from hazelcast.jar -->
+        <file>
+            <source>${configs.from.jar}/META-INF/NOTICE</source>
```

Comment:
We could rename this property to `files.from.jar` but that's really nit picking :-)